frameworks/JavaScript/hapi/start-servers.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+CPU_COUNT=$(nproc)
+P=3000
+END=$(($P+$CPU_COUNT))
+
+while [ $P -lt $END ]; do
+  PORT=$P node create-server.js &
```

Comment:
In this loop, you could use `sed` to insert your `server` lines under `upstream workers {` in your `nginx.conf` or use a separate script. Either way, that has to be created dynamically as well.

Reply:
ah, right. Thank you for pointing that out